Makefile

```diff
@@ -98,7 +98,7 @@ open-proxy:
 DB_DATA_DIR ?= container-data/db
 DB_CONTAINER_NAME ?= prod-kscout-bot-api-db
 DB_USER ?= prod-kscout-bot-api
-DB_PASSWORD ?= secretpassword
+DB_PASSWORD ?= ${DB_PASS}
```

Comment:
What is the reason behind this change? Couldn't users just set `DB_PASSWORD` before. Now they have to set `DB_PASS` to set `DB_PASSWORD`?